OAuth/ResourceOwner/FacebookResourceOwner.php

```diff
@@ -45,6 +47,14 @@ public function getUserInformation(array $accessToken, array $extraParameters =
         return parent::getUserInformation($accessToken, $extraParameters);
     }
 
+    /**
+     * {@inheritdoc}
+     */
+    public function getProfilePicture()
```

Comment:
This method doesn't belong here, it should only be part of `FacebookUserResponse`.

Reply:
@XWB Yep, it's an error, my bad.